src/strtree.c

```diff
@@ -16,6 +16,38 @@
 #include "pygeom.h"
 #include "kvec.h"
 
+/* GEOS function that takes a prepared geometry and a regular geometry
+ * and returns bool value */
+
+typedef char FuncGEOS_YpY_b(void *context, const GEOSPreparedGeometry *a,
+                            const GEOSGeometry *b);
+
+
+/* Copy values from arr to a new numpy integer array.
+ * The order of values from arr is inverted, because arr is created by pushing
+ * values onto the end. */
+
+static PyArrayObject *copy_kvec_to_npy(npy_intp_vec *arr)
+{
+    npy_intp i;
+    npy_intp size = kv_size(*arr);
+    npy_intp *ptr;
+
+    npy_intp dims[1] = {size};
+    // the following raises a compiler warning based on how the macro is defined
```

Comment:
@jorisvandenbossche I flagged the same compiler error you found.  I did a bit of digging into it, and couldn't resolve; it appears to come from `numpy` this way.

see [this](https://stackoverflow.com/a/32414359) and [this](https://stackoverflow.com/a/40900812)

I believe the compiler warning was there originally?